Source/JavaScriptCore/wasm/WasmBBQJIT64.cpp

```diff
@@ -2244,6 +2244,7 @@ PartialResult WARN_UNUSED_RETURN BBQJIT::addStructGet(ExtGCOpType structGetKind,
     TypeKind resultKind = structType.field(fieldIndex).type.unpacked().kind;
     if (structValue.isConst()) {
         // This is the only constant struct currently possible.
+        // If we add more constant structs, we need to update emitMoveConst
```

Comment:
I think rather than updating `emitMoveConst`, which is a general utility function, we should handle this case locally - see other cases like `BBQJIT::load` for examples of us materializing a zero value inline. It's probably also acceptable for us to literally just do `result = topValue(resultKind)` in this constant case as well, so we don't waste any space materializing a constant that will never be used.